api/rest/participants.py

```diff
@@ -0,0 +1,67 @@
+from datetime import date
+from rest_framework import serializers, status, validators, viewsets
+from rest_framework.response import Response
+
+from ..models import Participant, Team
+
+
+def is_true(value):
+    if not value:
+        raise serializers.ValidationError('Field is required')
+
+
+def is_eighteen(value):
+    today = date.today()
+    age = today.year - value.year - ((today.month, today.day) < (value.month, value.day))
+
+    if age < 18:
+        raise serializers.ValidationError('Must be older than 18 years')
+
+
+class ParticipantSerializer(serializers.ModelSerializer):
+    email = serializers.EmailField(validators=[
+        validators.UniqueValidator(queryset=Participant.objects.all())
```

Comment:
Email should be unique not only among `Participant`s, bat also among `auth.User`s and in LDAP. Although that should be checked in model, because we doesn't want to create email duplicity in admin too.